lib/debootstrap.sh

```diff
@@ -211,7 +211,7 @@ create_rootfs_cache()
 
 		# stage: update packages list
 		display_alert "Updating package list" "$RELEASE" "info"
-		eval 'LC_ALL=C LANG=C chroot $SDCARD /bin/bash -c "apt -q -y $apt_extra update"' \
+		eval 'LC_ALL=C LANG=C chroot $SDCARD /bin/bash -c "apt-get -q -y $apt_extra update"' \
```

Comment:
Chroot apt does not need changes.

Reply:
I replaced them all just to make sure...

Reply:
Fixed those. Let apt remain there.